api.swagger.yaml

```diff
@@ -5406,6 +5406,115 @@ paths:
+        '200':
+          description: 'The content was saved successfully.'
+          headers:
+            Location:
+              description: 'The URL at which the newly saved content may be retrieved.'
+              schema:
+                type: string
+        '400':
+          description: 'The content was not saved due to malformed JSON.'
+        '403':
+          description: 'User is not authorized to save a content.'
+        '422':
+          description: 'The content was not saved due to Drupal validation failure or JSON schema validation failure. (Contents may be closed for this content)'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                value:
+                  description: 'Flag to set'
+                  type: string
+                  example: '1'
+                value_type:
+                  description: 'type of value'
+                  type: string
+                  example: 'points'
+                entity_type:
+                  description: 'type of entity'
+                  type: string
+                  example: 'node'
+                tag:
+                  description: tag
+                  type: string
+                  example: 'plus1_node_vote'
+        required: true
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/saving:write ]
+    delete:
+      tags:
+        - saving
+      description: 'Delete one content.'
+      responses:
+        '200':
+          description: 'The content was un-saved.'
+          headers:
+            Location:
+              description: 'The URL at which the un-saved content may be retrieved.'
+              schema:
+                type: string
+        '400':
+          description: 'The content was not un-saved due to malformed JSON.'
+        '403':
+          description: 'User is not authorized to un-save a content.'
+        '422':
+          description: 'The content was not un-saved due to Drupal validation failure or JSON schema validation failure. (Contents may be closed for this content)'
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/saving:write ]
+    get:
+      description: Check if the content is saved or not.
+      tags:
+        - saving
+      parameters:
+        - name: content_id
+          description: The Invotra UUID of content.
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: '01234567-89ab-cdef-1234-56789abcdef0'
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/saving:read ]
+      responses:
+        '200':
+          description: Returns true/false if the content is saved or not.
+          content:
+            text/html:
+              schema:
```

Comment:
1) schema should be referenced externally to the response ('components' in the swagger)
2) Can we go with an object for this schema in case it needs expansion later (you'll notice we do this all over the place for future proofing) e.g.

{
  "current_user_saved": true";
}